agent/config/config.go

```diff
@@ -297,6 +301,25 @@ func environmentConfig() Config {
 		seelog.Warnf("Invalid format for \"ECS_NUM_IMAGES_DELETE_PER_CYCLE\", expected an integer. err %v", err)
 	}
 
+	instanceAttributesEnv := os.Getenv("ECS_INSTANCE_ATTRIBUTES")
+	attributeDecoder := json.NewDecoder(strings.NewReader(instanceAttributesEnv))
+	var instanceAttributes map[string]string
+
+	err = attributeDecoder.Decode(&instanceAttributes)
+	if err != io.EOF && err != nil {
+		err := fmt.Errorf("Invalid format for ECS_INSTANCE_ATTRIBUTES. Expected a json hash")
+		seelog.Warn(err)
+		errs = append(errs, err)
+	}
+	for k := range instanceAttributes {
```

Comment:
nit: Can you please use a long, more meaningful name here? How about `attributeKey`? Could probably use `key, value := range map` syntax as well.